config/example-extras.cfg

```diff
@@ -637,13 +637,13 @@
 #stow_on_each_sample: True
 #   If set to False, intermediate STOW/DEPLOY sequences on certain
 #   operations that use the probe multiple times will be omitted.
-#set_ouput_mode: none
-#   Possible values: 5V, 0V or None (default)
+#set_ouput_mode: [Possible values: 5V, 0D, or do not specify this
```

Comment:
Extra nit-picky maybe, but it looks like this is supposed to be "OD" but appears to say "0D" (zero instead of oh). I know it isn't affecting functionality but it might be confusing to someone looking at the supported options.

Reply:
Arrrgh